sdcm/tester.py

```diff
@@ -72,7 +72,8 @@
 from sdcm.utils.common import format_timestamp, wait_ami_available, update_certificates, \
     download_dir_from_cloud, get_post_behavior_actions, get_testrun_status, download_encrypt_keys, PageFetcher, \
     rows_to_list, make_threads_be_daemonic_by_default, ParallelObject, clear_out_all_exit_hooks, \
-    change_default_password, get_partition_keys
+    change_default_password
+from sdcm.utils.table_data import PartitionsValidationAttributes, get_partition_keys, TOTAL_ROWS_NUMBER_LIMIT
```

Comment:
`sdcm.utils.table_data` is not a good name for it... at least, it doesn't "click" anything to me by this name... maybe it should be `table_data_validation`?

Reply:
renamed to `table_metadata.py`